latexdiff

```diff
@@ -2855,7 +2862,8 @@ sub postprocess {
       ###if ( defined($packages{"listings"} and $latexdiffpreamble =~ /\\RequirePackage(?:\[$brat0\])?\{color\}/))   { 
 	# mark added verbatim commands
       $addblock =~ s/\\DIFverb/\\DIFDIFaddverb/g;
-      $addblock =~ s/\\DIFlstinline/\\DIFDIFaddlstinline/g;
+      1 while $addblock =~ s/((?<!\\)%.*)\\DIF($LSTINLINEENV)/$1\\DIFDIFadd$2/mg;
```

Comment:
I probably overlook something simple but why did you add the line `1 while $addblock ...` . 
As far as I can tell without testing this will change \DIFlstinline (and variants) in comments into \DIFDIFaddlstinline.
But the following line makes this change everywhere (including comments).

Reply:
You are right, I missed to remove this line. Commented inline-code is handled at 3 other places now.